### PR TITLE
DevEx 889/663: Remove case migration code

### PR DESCRIPTION
**Pull request by sutt0n**
Addresses devex cards 663 and 889

### CODE REVIEW

File: web-client/integration-tests/caseMigrations.test.js [hunk 1]
```diff
@@ -1,607 +0,0 @@
...
-  it('Docketclerk views case with casedeadlines', async () => {
-    await test.runSequence('gotoCaseDetailSequence', {
-      docketNumber: otherFilersCase.docketNumber,
-    });
-    expect(test.getState('caseDeadlines').length).toBe(1);
-  });
-
-  loginAs(test, 'petitioner@example.com');
-
-  it('user with e-access should see migrated e-access case on their dashboard', async () => {
```
Comment (rachaelparris) [inline web-client/integration-tests/caseMigrations.test.js:588]: I wonder if some of this code that was removed is still coverage that we should keep (because there still is migrated data on the site and we don't want to break its functionality), but we just need to create the test data differently since we can't hit the endpoint directly now that it's been removed.

File: web-api/src/app.js [hunk 2]
```diff
@@ -778,7 +777,6 @@ app.post(
 /**
  * migrate
  */
-app.post('/migrate/case', lambdaWrapper(migrateCaseLambda));
 app.post('/migrate/case-deadline', lambdaWrapper(migrateCaseDeadlineLambda));
 app.post('/migrate/trial-session', lambdaWrapper(migrateTrialSessionLambda));
```
Comment (rachaelparris) [inline web-api/src/app.js:788]: I bet we can get rid of these two as well (can be in another PR!)

Comment (sutt0n) [reply web-api/src/app.js:788]: Yeah, there was a lot on this PR alone, didn't wanna remove too much at once :)

File: web-client/integration-tests/externalUserViewsLegacySealedDocuments.test.js [hunk 3]
```diff
@@ -1,120 +0,0 @@
...
-
-  it('should migrate a case with a isLegacySealed docket entry', async () => {
-    await axiosInstance.post(
-      'http://localhost:4000/migrate/case',
-      caseWithEAccess,
-    );
-  });
-
-  loginAs(test, 'petitioner2@example.com');
-  associatedUserViewsCaseDetailForCaseWithLegacySealedDocument(test);
```
Comment (rachaelparris) [inline web-client/integration-tests/externalUserViewsLegacySealedDocuments.test.js:97]: This is also coverage that I think is really important to keep in here, since the legacy data does still exist in the system and we don't want to break this functionality.

Comment (rachelschneiderman) [reply web-client/integration-tests/externalUserViewsLegacySealedDocuments.test.js:97]: I agree. If you want more eyes, I am down to help with this!

**Review by rachaelparris — APPROVED** [review]
